code/API_definitions/Call_Forwarding_Signal.yaml

```diff
@@ -49,10 +51,21 @@ info:
     This is the resource the API Consumer uses to define the phone number to
     be verified about the status of the Network Call Forwarding service.\
     \
+    **UnconditionalCallForwardingSignal**
+    This is the resource the API Consumer gets back, containing the information
+    about the Unconditional Call Forwarding Service status for the given phone
+    number (PhoneNumber)\
+    \
     **CallForwardingSignal**
-    This is the resource the API Producer provides back to the API Consumer.
-    This Resource contains the status of the Network Call Forwarding service for
-    the specified phone number.
+    This is the resource the API Consumer gets back, containing the information
+    about the general status of the Network Call Forwarding service for the
+    specified phone number.\
+    \
+    The CFS API provides two endpoints fulfilling the following intents:
+    - **unconditional-call-forwardings**: Is the unconditional call fwd service
+    active on a specific phone number?
+    - **call-forwardings**: Provide me the status of the call FWD Service on a
```

Comment:
As we use interrogative form for unconditional-call-forwardings we should do same for call-forwardings for consistency. "What is the status of the call forwarding for a specific phone number?"

Reply:
I had the same doubt myself.

I have just modified it: https://github.com/camaraproject/CallForwardingSignal/discussions/6#discussioncomment-9327549

Reply:
maybe "Which is the...." ?

Reply:
Yeees ;)